variables.tf

```diff
@@ -6,15 +6,25 @@ variable "user_name" {
 variable "role_name" {
   description = "The name of the IAM role for the SFTP user. Either `role_name` or `role_arn` must be provided, not both."
   type        = string
-  default     = ""
 }
 
 variable "role_arn" {
-  description = "The name of the IAM role for the SFTP user. Either `role_name` or `role_arn` must be provided, not both."
+  description = "The ARN of a custom IAM role for the SFTP user."
```

Comment:
Perhaps this (and `role_name`, and maybe others) should have input validation instead of relying on the description?